tests/plugins/spf.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-const Address      = require('address-rfc2821').Address;
-const fixtures     = require('haraka-test-fixtures');
-const constants    = require('haraka-constants');
+const { Address }   = require('address-rfc2821');
+const fixtures      = require('haraka-test-fixtures');
+const constants     = require('haraka-constants');
```

Comment:
There's a lot of lines in this PR that have nothing changed except the addition or removal of a space character. That's churn and it's hard to argue it serves any useful purpose. It also makes reviewing the actual changes substantially more work. I'd love to see the superfluous whitespace changes reverted.